game/src/gui/app/state_ui.cc

```diff
@@ -352,6 +352,19 @@ bool StateUIUnitSelected::OnMouseButtonEvent(const foundation::MouseButtonEvent&
   return true;
 }
 
+bool StateUIUnitSelected::OnKeyEvent(const foundation::KeyEvent& e) {
+  if (e.IsKeyUp()) {
```

Comment:
Could you please add a line comment like `TODO Design a way to remap the key`? Not only this place but other OnKeyEvent handlers.

Reply:
OK